samples/scf-guess/input.dat

```diff
@@ -27,6 +27,9 @@ energy('scf')
 set guess huckel
 energy('scf')
 
+set guess sap
+energy('scf')
```

Comment:
Did you mean to add to the tests in `tests/scf-guess/` instead? This file is autogenerated from tests/ by stripping out the `#TEST` lines.

Reply:
I did.